addons/addon-base-raas/packages/base-raas-services/lib/study/__tests__/study-service.test.js

```diff
@@ -149,7 +149,7 @@ describe('studyService', () => {
       });
```

Comment:
I don't see a new test added checking the new branching logic that was added

Reply:
Done